controllers/users.js

```diff
@@ -15,17 +16,25 @@ const getNewUser = async (req, res, next) => {
       return res.status(409).json({ message: "Email is already used" });
     }
 
-    const newUser = new User({ email });
+    const verificationToken = nanoid();
+
+    const newUser = new User({ email, verificationToken });
     newUser.setPassword(password);
     await newUser.save();
```

Comment:
Jak wyslesz email to dopiero wtedy zapisz uzytkownika.

Reply:
Dobrym podejściem jest najpierw wysłanie emaila z tokenem aktywacyjnym, a następnie zapisanie tokenu do użytkownika po wysłaniu. Proces ten  minimalizuje ryzyko fałszywych rejestracji.